content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -156,6 +156,8 @@ export type QueryEvent = {
 }
 ```
 
+Note that for MongoDB, the `params` and `duration` fields will be undefined.
```

Comment:
I expect that type above is not correct any more then, correct @millsp?

Reply:
`NaN` is a number :trollface: but `undefined` is not a `string`, so that one is incorrect yes.

Reply:
What should I update it to?

Reply:
No worries, you can leave it as is because it is technically correct. We will need to fix the types after the release https://github.com/prisma/prisma/issues/12319